runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -243,7 +243,7 @@ TR_CISCNode::dump(TR::FILE *pOutFile, TR::Compilation * comp)
       {
       sprintf(buf, "%s", name);
       }
-   traceMsg(comp, "[%p] %3d %2d%c %-11s", this, _id, _dagId, isOutsideOfLoop() ? ' ' : 'L', buf);
+   traceMsg(comp, "[%p] %3d %2d%c %-11s", this, _id, _dagId, isOutsideOfLoop() ? ' ' : 'L', buf); //another unsure place for Q
```

Comment:
I'm guessing this is using 'L' as short for 'loop' in debug code, so I don't think this needs to handle `Q`.